microsoft/botframework-solutions
# Lamil/eb lu update

## Description

- Updated enterprise bot to use latest General LUIS model and include professional chitchat qna maker
- Update enterprise bot sample to include latest Telemetry and SDK version

## Related Issue




closes #357 

## Testing Steps


1. Copy "EnterpriseBotTemplate/Bot Framework" folder into your "Documents\Visual Studio 2017\Templates\ProjectTemplates" folder
2. Open VS and create new project using the template
3. Verify it builds with no errors
4. Run msbot clone deployment 
5. Run or publish project after deployment and verify the following:
   - Test Onboarding by clicking the "Get started" button on the intro card
   - Test Cancelling while in Onboarding by typing "Cancel"
   - Test LUIS using: "Help", "Talk to a human"
   - Test ChitChat using: "Hi", "excuse me", "will you marry me"
   - Test FAQ using: "What is the enterprise bot template?"

## Checklist
~~- [ ] I have commented my code, particularly in hard-to-understand areas~~
~~- [ ] I have added or updated the appropriate unit tests~~
   - Public docs PR here: https://github.com/MicrosoftDocs/bot-docs-pr/pull/1218

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
~~- [ ] A duplicate issue is filed to track future  work.~~

If you have updated responses or `.lu` files:
